state.c

```diff
@@ -99,6 +99,7 @@ struct pv_state* pv_state_new(const char *rev, state_spec_t spec)
 		dl_list_init(&s->jsons);
 		pv_state_init_groups(s);
 		dl_list_init(&s->conditions);
+		dl_list_init(&s->bsp.drivers);
```

Comment:
is it necessary to clean this list in pv_state_free? A change in bsp/drivers.json will always trigger a reboot update, but, maybe for completion's sake